src/main/flight/pid.c

```diff
@@ -1462,6 +1462,10 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
 #else
         const float Ki = pidCoefficient[axis].Ki;
 #endif
+        // iterm_windup on yaw only
+        if (axis < FD_YAW) {
```

Comment:
This relies on `FD_YAW` being the last axis that is processed in the loop, making it prone to accidental breakage.

Suggestion:

```
#ifdef USE_LAUNCH_CONTROL
        // if launch control is active override the iterm gains
        float Ki = launchControlActive ? launchControlKi : pidCoefficient[axis].Ki;
￼#else
￼        float Ki = pidCoefficient[axis].Ki;
￼#endif
￼        // iterm_windup on yaw only
        if (axis != FD_YAW) {
          Ki = Ki * dynCi;
        }
        pidData[axis].I = constrainf(previousIterm + (Ki + agGain) * itermErrorRate, -itermLimit, itermLimit);
```

Reply:
Thanks, Mike, much better.  
PS - I never knew if yaw was always last or not, now I do.

Reply:
It's called RPY. :wink: